ui/status.go

```diff
@@ -0,0 +1,62 @@
+package ui
+
+import (
+	"github.com/algorand/go-algorand-sdk/client/v2/algod"
+	"github.com/algorandfoundation/hack-tui/internal"
+	tea "github.com/charmbracelet/bubbletea"
+	"strconv"
+)
+
+// StatusModel is extended from the internal.StatusModel
+type StatusModel struct {
+	internal.StatusModel
+}
+
+// Init has no I/O right now
+func (m StatusModel) Init() tea.Cmd {
+	// Just return `nil`, which means "no I/O right now, please."
+	return nil
+}
+
+// Update is called when the user interacts with the render
+func (m StatusModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	switch msg := msg.(type) {
+
+	// Is it a key press?
+	case tea.KeyMsg:
+
+		// Cool, what was the actual key pressed?
+		switch msg.String() {
+
+		// These keys should exit the program.
+		case "ctrl+c", "q":
```

Comment:
I didn't realise bubbletea highjacks SIGINT and you have to explicitly define the behaviour. Whilst testing it I also thought I should include a contribution. 😄
```suggestion
		case "ctrl+c", "q", "esc":
```

Reply:
Now I'm thinking about it, maybe `esc` shouldn't exit the program, and it should just return you back to the main window (assuming modals or other views will eventually exist). Depends how the larger design looks I guess. Great work so far though!

Reply:
It's been a great experience! I really like what the baseline offers and where your thinking is heading. What the framework unlocks is pretty impressive!! 

I almost added the [editor integration](https://pkg.go.dev/github.com/charmbracelet/x/editor) for the config files and had to stop myself several times from diving down the rabbit hole 😂